src/Rules/Register.hs

```diff
@@ -83,6 +83,7 @@ buildConf _ context@Context {..} _conf = do
            , bldPath -/- "ghcautoconf.h"
            , bldPath -/- "ghcplatform.h"
            , bldPath -/- "ghcversion.h"
+           , bldPath -/- "ffi.h"
```

Comment:
Is this still needed? I might be misremember something here.

Reply:
Oh I didn't notice that I kept this around. Well, let's try removing it...

Reply:
The build breaks on my machine when I remove this line.